hooks/post_gen_project.py

```diff
@@ -1,9 +1,211 @@
+import os
+import json
+import shutil
 import subprocess
+from os.path import join
 
-if __name__ == "__main__":
-{% if cookiecutter.create_repo == "yes" %}
-    subprocess.check_call(['git', 'init'])
-    subprocess.check_call(['hub', 'create'])
-{%- else %}
+import git
+
+from python_terraform import Terraform
+
+try:
+    input = raw_input  # py2 compatibility
+except NameError:
     pass
-{% endif %}
+
+TF_MODULE_PATH = (
+    "git::https://github.com/plus3it/terraform-github-repo.git?ref=0.0.0")
```

Comment:
dilemma... if we put this source into a .tfvars file, specified similarly to how terragrunt specifies source, then dependabot will automatically update the ref for us... we'd just have to load the .tfvars file from hcl...

Reply:
I _think_ I have a workaround. The solution means that this `.tfvars` file has to live in the `{{cookiecutter.module_name}}` dir and that there's going to be an 'errant' `.gitingore` entry in the resulting repo(s), but not sure if there's a better way of dealing with this